src/rvsdg/egglog_optimizer/mod.rs

```diff
@@ -43,7 +43,7 @@ pub fn rvsdg_egglog_schedule() -> String {
         ; spine - we are working on this!), so we only run it a few times at the
         ; end to apply substitutions that the main optimizations find. It's
         ; interleaved with fast-analyses because it relies on reified vecs.
-        (repeat 6 subst-beneath (saturate fast-analyses))
+        (repeat 10 subst-beneath (saturate fast-analyses))
```

Comment:
Is it fast enough we can run this during normal rule execution, not just at the end?